### ISSUE TITLE
FFMPEG read camera stream on OSX

### ISSUE THREAD

**Issue by almarklein** [imageio/imageio#31]
Reading from a camera stream is supported on Linux and Win, but mot yet on OSX. Mostly because I cannot test it (or at least I expect accessing my camera from a VM will be problematic).

The main changes requires are in `_get_cam_inputname` to specify how to read the camera, and in the beginning of the module to set `CAM_FORMAT`. Would be good if video4linux would just work on osx, but I doubt that it would.

cc @rreilink (if you have time, someday)

**Comment by kuchi** [comment]
I got this to work with the pull request that I have, but I have to use my own FFMPEG binary. The one you have hosted does not seem to support video cameras on OS X.  The binary you have gives the error:

    Unrecognized option 'list_devices'.

**Comment by almarklein** [comment]
What version is your ffmpeg binary?

**Comment by almarklein** [comment]
Ah, not I see you wrote in the other issue "you do need to have ffmpeg installed through home-brew or some other means where it is correctly compiled for supporting cameras. Static builds for OS X on the web don't seem to support cameras."

Is this certain, or is it worth trying to update the lib. Otherwise we should put this information in the ffmpeg plugin documentation.

**Comment by kuchi** [comment]
The current version of the static ffmpeg binary you are hosting does not have support for avfoundation which is required on OS X to use cameras. I confirmed current static builds at https://ffmpeg.org/download.html#build-mac do work with the cameras but the -list-devices option to ffmpeg is missing/depreciated. However I updated the code yesterday to deal with that. So, if you just update to the above static build all should be good. The cameras work now on OS X. I can access my built in camera using <video0> without an issue.
